### ISSUE TITLE
[Suggestion] Make snack readable by screenreaders

### ISSUE THREAD

**Issue by phiter** [eolant/vuetify-toast-snackbar#23]
Hey!
I was creating my own solution and just came across yours, which seems to be more complete.

Screenreaders aren't reading the toast notification, that's because it lacks a `role="alert"` on it.

Just putting `role="alert"` on the `<v-snackbar` fixes the issue.

**Comment by eolant** [comment]
@phiter Thanks for the suggestion, I'll add it.
